ql/src/java/org/apache/hadoop/hive/ql/exec/Utilities.java

```diff
@@ -4272,6 +4273,21 @@ public static void addTableSchemaToConf(Configuration conf,
       LOG.info("schema.evolution.columns and schema.evolution.columns.types not available");
     }
   }
+  public static void setPartitionColumnsToConf(Configuration conf, TableScanOperator tableScanOp) {
```

Comment:
nit: pls add a comment for this method.  Also, the unset method is named 'InConf' whereas this one has 'ToConf'.  Can both set and unset method names use the same suffix ?

Reply:
Setting both to InConf.